test/views/NotificationsViewSpec.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewValidator
+import org.scalatest.{MustMatchers, WordSpec}
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import play.twirl.api.Html
+import utils.Stubs
+import views.html.notifications
+
+class NotificationsViewSpec extends WordSpec with MustMatchers with Stubs with ViewValidator {
+
+  val messages = stubMessages()
+  val page: Html = new notifications(mainTemplate)(Seq.empty)(FakeRequest(), minimalAppConfig, messages)
+
+  "Notification page" should {
+
+    "contains title" in {
+
+      getElementById(page, "title").text() must be(messages("notifications.title"))
```

Comment:
Theres some nice matchers you can use for these e.g.

`page.getElementById("title") should containText(messages("notifications.title"))`

Reply:
This has been addressed.

Reply:
Apologies I wasn't saying what you'd done! It was just for future reference